symfony/psr-http-message-bridge/2.1/config/packages/psr_http_message_bridge.yaml

```diff
@@ -0,0 +1,20 @@
+services:
+    _defaults:
+        autowire: true
+        autoconfigure: true
+
+    Symfony\Bridge\PsrHttpMessage\HttpFoundationFactoryInterface:
+        class: Symfony\Bridge\PsrHttpMessage\Factory\HttpFoundationFactory
+
+    Symfony\Bridge\PsrHttpMessage\HttpMessageFactoryInterface:
+        class: Symfony\Bridge\PsrHttpMessage\Factory\PsrHttpFactory
```

Comment:
it'd be great to find a way to remove those two lines (see my comments on the PR on the bridge)

Reply:
Okay, let's keep the discussion on the bridge PR.

Reply:
You made a good point in the bridge PR: these are useful services on their own.
To really label them as useful services, the interfaces should be *aliases* to the actual implementation. This is the criteria that `debug:autowiring` needs to identify the "useful services".

Reply:
Both interfaces are aliases now. I've also changed the services in the comment and assigned service IDs to them.